src/c++/perf_analyzer/genai-perf/genai_perf/llm_metrics.py

```diff
@@ -161,14 +161,18 @@ class Statistics:
     def __init__(self, metrics: Metrics):
         # iterate through Metrics to calculate statistics and set attributes
         self._metrics = metrics
+        self._stats_dict: Dict = {}
         for attr, data in metrics.data.items():
             attr = metrics.get_base_name(attr)
+            self._stats_dict[attr] = {}
```

Comment:
If you use `defaultdict`, it will automatically create initial `dict` for you.
```suggestion
        self._stats_dict: Dict = defaultdict(dict)
        for attr, data in metrics.data.items():
            attr = metrics.get_base_name(attr)
```

Reply:
Learning as i go. thank you